extensions/configuration-editing/schemas/devContainer.schema.src.json

```diff
@@ -351,6 +351,27 @@
 									"type": "string",
 									"pattern": "^\\d+([tgmk]b)?$",
 									"description": "Amount of required disk space in bytes. Supports units tb, gb, mb and kb."
+								},
+								"gpu": {
+									"type": [
+										"boolean",
+										"string",
+										"object"
+									],
+									"pattern": "^optional$",
```

Comment:
Tried using `"enum": ["optional"]`, but that seemed to apply to all three types (boolean and object showed warnings in the editor).